.env.template

```diff
@@ -1,6 +1,6 @@
 SECRET_KEY=my-secret-key
 ENVIRONMENT=test
-DEBUG=False
+DEBUG=True
```

Comment:
I can't comment below but we also want to update `DB_HOST` and `DB_READONLY` as follows 

```env
DB_HOST=localhost
# ...
DB_READONLY=postgres://apilos:apilos@localhost:5433/apilos
# ...
#ECOLO_DATABASE_URL=postgres://apilos:apilos@localhost:5433/ecolotest?currentSchema=ecolo
``` 

Also, `SENDINBLUE_API_KEY` must be disabled by default to prevent error when sending emails 

```env
#SENDINBLUE_API_KEY=<SENDINBLUE_API_KEY>
```

Reply:
I have an issue about editing .env.template because it is used to test the application in CI (which is not the good solution :) )

Then I need to rollback the updates we done on this file.

What do you think, do we do it here or in another PR ?

Reply:
Now that I have a local running environment, I would say there is no rush in merging the current PR, and we could do fix this other issue in the current PR as it would benefit from both additions.

Reply:
👍 , will do a .env.citest file